front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm.tsx

```diff
@@ -180,6 +185,10 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
     []
   );
 
+  const handleProjectFolderImagesOnRemoveNoArgs = useCallback(() => {
+    handleProjectFolderImageOnRemove(projectFolderImages[0]);
+  }, [projectFolderImages]);
+
```

Comment:
I don't think this useCallback does something useful (nor is needed). Also, in this file many functions are wrapped with useCallback and an empty dependencies array, which is the same as not wrapping with useCallback at all. So let's start cleaning up this file by removing all these?

Reply:
In fact, no function in this file needs useCallback. They can all be removed.